mscore/pa.h

```diff
@@ -67,7 +69,7 @@ class Portaudio : public Driver {
       int deviceIndex(int apiIdx, int apiDevIdx);
       int currentApi() const;
       int currentDevice() const;
-      MidiDriver* mididriver() {return midiDriver;}
+      MidiDriver* mididriver() {return midiDriver;}      
```

Comment:
thanks, jojo...I've repushed to fix that.  I was hoping for some higher-level comments.